### DISCUSSION TITLE
video slide works incorrect with cube effect

### DISCUSSION THREAD

**Discussion by yuuk** [nolimits4web/swiper#6060] [Q&A]
### Check that this is really a bug

- [x] I confirm

### Reproduction link

https://codesandbox.io/s/swiper-with-image-and-video-xzt835?file=/index.html

### Bug description

I have a swiper with image slide and video slide, the height of them are different, when I swipe the video slide to the image slide, the previous video slide is still visible.

ps: I've tested v6.8.4 and 8.3.2

![Snipaste_2022-08-06_22-07-40](https://user-images.githubusercontent.com/9426456/183252514-56955d56-56cc-44b8-ba82-c21a487507fc.jpg)

### Swiper version

v6.8.4 and 8.3.2

### Platform/Target and Browser Versions

windows chome 103.0.5060.134

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [X] I'm willing to open a PR

**Comment by dijkermans** [comment]
I think you need to set a fixed height with overflow:hidden

It only seems to work fine with images/videos with the exact same ratio.
